liquidprompt

```diff
@@ -1137,6 +1140,27 @@ _lp_jobcount_color() {
     [[ -n "$lp_jobcount_color" ]]
 }
 
+###################
+# Directory stack #
+###################
```

Comment:
This doesn't need its own section, I would put it under the Environment section (line 834).